### PR TITLE
Add quick-link for users coming from UI

### PR DESCRIPTION
**Pull request by lkysow**
The Consul UI topology view has an icon with the text
"Configure metrics dashboard" that links to this page. Add a notice at
the top of the page that links them directly to the relevant section.

### CODE REVIEW

File: website/content/docs/connect/observability/ui-visualization.mdx
Comment (johncowen) [inline website/content/docs/connect/observability/ui-visualization.mdx:11]: 😆 do we want to change this to "Configure dashboard" now?

**Review by johncowen — APPROVED** [review]
Left one comment which you may or may not want to address, but other than that LGTM!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/492332.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9b680b3f814ab544f5af9491ed3017cf0ce5d886 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9b680b3f814ab544f5af9491ed3017cf0ce5d886 onto `release/1.10.x` succeeded!

**Pull request merged**
